crates/rolldown/src/module_loader/module_task.rs

```diff
@@ -77,6 +77,12 @@ impl ModuleTask {
   #[tracing::instrument(name="NormalModuleTask::run", level = "trace", skip_all, fields(module_id = ?self.resolved_id.id))]
   pub async fn run(mut self) {
     if let Err(errs) = self.run_inner().await {
+      self
+        .ctx
+        .plugin_driver
+        .mark_context_load_modules_loaded(&ModuleId::new(&self.resolved_id.id), false)
+        .await
+        .expect("Mark context load modules loaded should not fail");
```

Comment:
Use another panic message or use `unwrap` directly. This message looks weird when it is a `Err` variant; anything wrapped with `Result` could be a `Err`

Reply:
The error only happened at call js side function failed, the function is the internal implement, the message maybe is wired for the users, it only used to the developers, it could be clear for me.